python/requirements.txt

```diff
@@ -6,4 +6,4 @@ click==6.7
 itsdangerous==0.24
 requests==2.20.0
 wsgiref==0.1.2 ; python_version < '3.2'
-plaid-python>=3.0.0
+plaid-python>=5.0.0, < 6.0.0
```

Comment:
why not >= `6.0.0` here?

Reply:
<img width="833" alt="image" src="https://user-images.githubusercontent.com/38868412/88094440-9462f180-cb61-11ea-9339-99ed9515ea25.png">

Reply:
LOL noted